### ISSUE TITLE
`hint inputs --last`

### ISSUE THREAD

**Issue by tkna91** [qutebrowser/qutebrowser#7303]
I am using DeepL and often want to focus on the right input field to copy text for back translation for checking.
![image](https://user-images.githubusercontent.com/102382754/178135906-df0d5991-0330-412a-8ef8-8427a13b3f05.png)

Can you make something like the following command?

`hint-inputs --last`

Or is there some sort of alternative?

**Comment by The-Compiler** [comment]
In this particular case, why not just use the "swap" button in the middle instead?

In a more general sense, we have `:click-element`, recently also extended by CSS selectors (#6609), to click specific elements. I guess we could extend hints in the way you propose, but I wonder if it wouldn't be a bit too flaky, as the position of hints can change a lot depending on what exactly is on the screen.

**Comment by tkna91** [comment]
Thank you.
I am copy-pasting manually because the undo buffer disappears when I press the swap button and I cannot re-edit the translation source.

I am working on the following.
https://qutebrowser.org/doc/help/commands.html#click-element

Do you have any kind of example? Sorry I can't find it well.

Commands executed

```
:click-element filter css "lmt__textarea lmt__textarea_dummydiv"
```

Probably the source around what is being targeted.

```
<textarea aria-labelledby="translation-results-heading" class="lmt__textarea lmt__target_textarea lmt__textarea_base_style" data-gramm_editor="false" dl-test="translator-target-input" autocomplete="off" lang="ja-JP"></textarea>
```

**Comment by The-Compiler** [comment]
Your `filter` argument should just be `css` - and classes need a `.` for CSS selectors, e.g. `.lmt__textarea`.

However, that will find multiple elements. I'd suggest something like `:click-element css "textarea[dl-test=translator-target-input]"` based on the `dl-test` attribute they set, which hopefully is stable enough.

**Comment by tkna91** [comment]
I did the following, but it still doesn't seem to work. I would appreciate it if you could take a look at it when you are free. Sorry for the trouble.

- `click-element css "textarea[dl-test=translator-target-input]"`
- `click-element css 'textarea[dl-test="translator-target-input"]'`
- `click-element css textarea"[dl-test='translator-target-input']"`
- `click-element css .lmt__textarea`
![20220711-211514_screenshot](https://user-images.githubusercontent.com/102382754/178262667-dc44dfd6-4a60-4c36-a185-aecec55f03f6.png)
https://www.deepl.com/translator

**Comment by The-Compiler** [comment]
#6609 was only merged recently and is not released yet, you need to run from git master to use it.

**Comment by tkna91** [comment]
I was able to do it. Thank you!

```
$ paru -Q qutebrowser-git
qutebrowser-git 2.5.2.r378.g63caa39a2-1
$
```
- `click-element css "textarea[dl-test=translator-target-input]"`

https://user-images.githubusercontent.com/102382754/178272280-a99e93c7-b5c2-4a29-8430-7272fe15f397.mp4

**Comment by tkna91** [comment]
I am sorry to have just received your solution, but in addition to that, I feel that it would be useful to have something like the command I suggested at the beginning of this article that could be used on any site.
If it is reliably hard to work, it would be nice to indicate so to users in help, etc.
How do you feel about this?

**Comment by tkna91** [comment]
Certainly, the order of up, down, left, right, and right that user judge from the appearance may be different from the order in the source, etc. If the order is the same as the order in which the hints are displayed, then perhaps such an explanation should be included in the help, etc.
